ports/stm32/uart.c

```diff
@@ -577,6 +591,10 @@ STATIC void pyb_uart_print(const mp_print_t *print, mp_obj_t self_in, mp_print_k
             self->uart.Init.StopBits == UART_STOPBITS_1 ? 1 : 2,
             self->timeout, self->timeout_char,
             self->read_buf_len == 0 ? 0 : self->read_buf_len - 1); // -1 to adjust for usable length of buffer
+        if (self->irq.enable != IRQ_FLAG_NONE) {
+            mp_printf(print, " IRQ callback on%s",
+                      (self->irq.enable&IRQ_FLAG_RX_IDLE)?" RX_IDLE_IRQ":"");
+        }
```

Comment:
A lot simpler to just print out (in hex) the value of `self->irq.enable`.  Then users can find out the actual flag names by inspecting the values of the constants like `RX_IDLE_IRQ`.